src/compositions/message/message.tsx

```diff
@@ -55,6 +55,13 @@ export default function Message({ message, displayVariant, children }: MessagePr
         'z-5 relative row-span-3', // mobile
         'md:absolute md:left-[-32px] md:top-[20px]', // desktop
     );
+    const shortTextOfPost = clsx(
+        'break-all text-secondary-900',
+        isVariant(MessageDisplayVariant.DETAIL_VIEW) || isVariant(MessageDisplayVariant.INLINE)
```

Comment:
`isVariant(MessageDisplayVariant.TIMELINE)` instead of `isVariant(MessageDisplayVariant.DETAIL_VIEW) || isVariant(MessageDisplayVariant.INLINE)`